makefile

```diff
@@ -34,6 +45,61 @@ stop:
 	@ docker-compose -f $(DOCKER_COMPOSE_FILE) down -v
 	@ echo "All containers stopped successfully"
 
+build_backend:
```

Comment:
Do we have a readme doc for these changes?

Reply:
ReadMe doc updated.